google-cloud-storage/acceptance/storage_helper.rb

```diff
@@ -105,18 +106,17 @@ def self.run_one_method klass, method_name, reporter
   end
 end
 
-def safe_gcs_execute retries: 20, delay: 2
-  current_retries = 0
-  loop do
-    begin
-      return yield
-    rescue Google::Cloud::ResourceExhaustedError
-      raise unless current_retries >= retries
-
-      sleep delay
-      current_retries += 1
-    end
+def safe_gcs_execute retries: 5
+  max_retries = 10
```

Comment:
This is to guard the users of this function from accidentally doing too many retries. Since retry delays increase exponentially, the time-per-test can go very high.